ObsCoreExtensionForRadioData.tex

```diff
@@ -98,45 +99,74 @@ \section{Introduction}
 
-\subsection{Single dish and beam forming data}
+Contrary to what usually happens for  interferometric observations, in the case of INAF radio telescopes a SD observation (scan) contains only one scientific target. In any case, each target in an observation is listed as a separate entry in an ObsCore Table sharing the same obs\_id.
 
-We list here all the radio observation receivers and modes which provide direct measurements of fluxes for some positions, frequencies, polarization and time stamps.
-In the simplest case Single Dish data gather emission in a central beam for each spectral sample in the spectral band and polarization and for some time stamps. They can provide either spectra, time series or dynamical spectra ( the case where observers span the time axis, record time stamps and thus produce dynamical spectra is however tackled in an other document). 
+Complex frequency setups are possible in the same observation, as already mentioned in Sect. \ref{sec:specificities}. For instance, new multi-band receivers acquire simultaneously different frequency bands with different setups (bandwidth, max/min frequency, etc.). Moreover, modern digital backends allow the simultaneous acquisition of more than one spectral window (each with its spectral resolution) inside each receiver band. Such a complex observation should be properly represented in an ObsCore Table.
+As an example: the new tri-band K-Q-W receiver being installed on board of the INAF radio telescopes observes simultaneously the K, Q and W frequency bands in the same observations. It can be coupled with a digital backend delivering {\it n} different spectral windows for each of the three observed bands. In this example, if the highest-granularity approach to ObsCore is applied, such a case would be described with $3n$ entries in a ``flat'' ObsCore Table. In case of broadband spectro-polarimetric observations, like the simultaneous study of many emission lines, $n$ may become large. Thus, such numerous ObsCore entries associated to the same dataset might make it difficult for a user to understand the observing setup. The possibility to adopt different levels of granularity in an ObsCore Table might be considered.
 
-However in many cases more sophisticated observing modes allow to span wider parts of the sky. Fig \ref{fig:SD} shows some of these modes. 
+The ObsCore definition of t\_resolution as the minimal interpretable interval between two points along the
+time axis (being it an average or representative value) is generally not applicable for SD data. Typically time is not an independent variable in SD measurements, it can be saved together with spatial/spectral/intensity information as a timestamp associated to each data sample.
+Even in the case of on-source tracking, time information in SD data is not intended for time domain studies.
```

Comment:
The duration is referred to the integration time used to sample data. The spatial position is typically different among integration intervals, since the telescope is typically moving during the observation (sky scanning). Even in the case in which the telescope is observing a fixed sky position, the integration interval is not related to some kind of quantity that must be time-resolved. In most cases, you may change the value of the integration time interval without affecting the scientific result provided you increase the total observation duration. The scientific content in a single time interval is not meaningful by itself, usually. Maybe we can discuss this further.